src/query_helpers/question_validator.py

```diff
@@ -18,7 +18,7 @@ def __init__(self):
         """
         self.logger = Logger("question_validator").logger
 
-    def validate_question(self, conversation, query, **kwargs):
+    def validate_question(self, conversation: str, query: str, **kwargs) -> list[str]:
```

Comment:
Does this return a list of strs or just a single str?

Reply:
list/array: https://github.com/openshift/lightspeed-service/pull/81/files#diff-9ca3ebc421243362c6b59edda5939229ef8cd3c3cd4c83753986de3b315d3092L66